main.sh

```diff
@@ -401,12 +401,12 @@ versionSelector() {
 }
 
 checkPatched() {
```

Comment:
I'm guessing some logic is needed in this function (lines 403-423) to check for the old file name format (and rm it if it exists) to allow reverse compatibility, but I'm not 100% sure